src/components/Header/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Header.css';
+
+export default function Header() {
+  return (
+    <div className="header">
+      <NavLink className="nav" exact to="/">
+        Home
+      </NavLink>
+      <NavLink className="nav" exact to="/teams">
+        Teams
+      </NavLink>
+      <NavLink className="nav" exact to="/players">
+        Players
+      </NavLink>
+    </div>
```

Comment:
Perfect execution here! To prevent the same active class from rendering the same colors, we can add styling for a link and styling for an active link and have different colors for both.